test_conformance/math_brute_force/binary_double.cpp

```diff
@@ -20,10 +20,12 @@
 
 #include <cstring>
 
-const double twoToMinus1022 = MAKE_HEX_DOUBLE(0x1p-1022, 1, -1022);
+namespace {
 
-static int BuildKernel(const char *name, int vectorSize, cl_uint kernel_count,
```

Comment:
In one of the [CI runs](https://ci.appveyor.com/project/Khronoswebmaster/opencl-cts/builds/39199261/job/rs6ku4phah8he58q) on windows seeing

```
 C:\projects\opencl-cts\test_conformance\math_brute_force\binary_double.cpp(25): error C2131: expression did not evaluate to a constant 
```

and for `binary_float.cpp(25)`

Reply:
Yes, indeed. I reverted the const to constexpr change. diff here: https://github.com/KhronosGroup/OpenCL-CTS/compare/b3f457d13e5801371403f1039ddac75d655f3e1b..415c2482cc53bc5973db4de148b5f363c78084e3

Reply:
Thanks, change looked good otherwise